pkg/daemon/ceph/osd/agent.go

```diff
@@ -136,7 +136,7 @@ func (a *OsdAgent) removeDirs(context *clusterd.Context, removedDirs map[string]
 
 	if len(failedDirs) > 0 {
 		// at least one OSD failed, return an overall error
-		return fmt.Errorf("failed to cleanup directories: %v", failedDirs)
+		return fmt.Errorf("failed to cleanup directories: %+v", failedDirs)
```

Comment:
`%v` and `%+v` print the same thing for basic string collection types: `[]string` and `map[string]string`. This and several other of the changes in this PR won't give you more info. For these string collection types I typically will just us `%v`, but either way is fine too.

Reply:
I made several changes individually from `%v` to  `%+v` when data structures were being printed without field names before I just did a copy-replace on all of them. So definitely all of them aren't necesarry, but it seemed easier to do all of them than to search manually. I think I'm inclined to let them all be `%+v` for the ease here, but I don't want to churn the code too much if there is concern about that.

Reply:
that's fine to keep it, no problem.